### PR TITLE
chore: update members of Russian's localization team

### PR DESCRIPTION
**Pull request by leon-win**
### Description

This PR updates list of Russian's localization team members.
I've added myself :sunglasses: and rearranged the list in order of current availability.

### Related issues and pull requests

Relates to #17071, https://github.com/mdn/content/pull/30662

### CODE REVIEW

File: files/zh-cn/mdn/community/contributing/translated_content/index.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ slug: MDN/Community/Contributing/Translated_content
 ### 俄语（ru）
 
 - 讨论组：[Matrix (#mdn-l10n-ru channel)](https://chat.mozilla.org/#/room/#mdn-l10n-ru:mozilla.org)
-- 目前的志愿者：[yanaklose](https://github.com/yanaklose)、[myshov](https://github.com/myshov)、[Saionaro](https://github.com/Saionaro)、[sashasushko](https://github.com/sashasushko)、[lex111](https://github.com/lex111)
+- 目前的志愿者：[leon-win](https://github.com/leon-win), [sashasushko](https://github.com/sashasushko), [Saionaro](https://github.com/Saionaro), [yanaklose](https://github.com/yanaklose), [myshov](https://github.com/myshov), [lex111](https://github.com/lex111)
```
[the remark this answers is not in the document]
Comment (leon-win) [reply files/zh-cn/mdn/community/contributing/translated_content/index.md]: Sorry, `copy-paste` played a cruel joke on me(
But now I’ll remember how to use commas correctly in Chinese))
Thank you @yin1999!

**Review by yin1999 — APPROVED** [review]
LGTM for zh-CN. Welcome 🎉

**Review by SphinxKnight — APPROVED** [review]
✔️ for `fr` :)

**Comment by mfuji09** [comment]
`ja` looks okay!

**Review by Graywolf9 — APPROVED** [review]
LGTM for `es`, be very welcome and thank you for all your help!

**Review by dipikabh — APPROVED** [review]
Welcome aboard. Thanks for doing this important community work!

**Pull request merged**